sql-plugin/src/main/330+/scala/com/nvidia/spark/rapids/shims/v2/Spark33XShims.scala

```diff
@@ -153,6 +157,201 @@ trait Spark33XShims extends Spark33XFileOptionsShims {
     }
     super.tagFileSourceScanExec(meta)
   }
+
+  // 330+ supports YEARMONTH and DAYTIME interval types
+  override def getFileFormats: Map[FileFormatType, Map[FileFormatOp, FileFormatChecks]] = {
+    Map(
+      (ParquetFormatType, FileFormatChecks(
+        cudfRead = (TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 + TypeSig.STRUCT +
+            TypeSig.ARRAY + TypeSig.MAP + TypeSig.DAYTIME).nested(),
+        cudfWrite = (TypeSig.commonCudfTypes + TypeSig.DECIMAL_128 + TypeSig.STRUCT +
+            TypeSig.ARRAY + TypeSig.MAP + TypeSig.DAYTIME).nested(),
+        sparkSig = (TypeSig.cpuAtomics + TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP +
+            TypeSig.UDT + TypeSig.DAYTIME).nested())))
+  }
+
+  // 330+ supports YEARMONTH and DAYTIME interval types
+  override def getExprs: Map[Class[_ <: Expression], ExprRule[_ <: Expression]] = {
+    val _gpuCommonTypes = TypeSig.commonCudfTypes + TypeSig.NULL + TypeSig.DECIMAL_64
```

Comment:
Same here, everywhere this is used the DECIMAL_64 is replaced with a DECIMAL_128

Reply:
Updated.